presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1511,13 +1587,23 @@ schema evolution, such as adding, dropping, and renaming columns. With schema
 evolution, users can evolve a table schema with SQL after enabling the Presto
 Iceberg connector.
 
+Presto C++ Support
```

Comment:
This is for Partition Column Transform. Could you please add Presto C++ support as well? Thanks!